air/src/air/coefficients.rs

```diff
@@ -32,6 +32,7 @@ pub struct ConstraintCompositionCoefficients<E: FieldElement> {
     pub transition: Vec<E>,
     pub boundary: Vec<E>,
     pub lagrange: Option<LagrangeConstraintsCompositionCoefficients<E>>,
+    pub s_col: Option<E>,
```

Comment:
Should we also update doc comments for this struct?

Reply:
Updated